src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts

```diff
@@ -17,11 +17,11 @@ import {IModalHost} from "../../../wizard/models/modal-host";
   styleUrls: ['./codebases-item-actions.component.less']
 })
 export class CodebasesItemActionsComponent implements OnDestroy, OnInit {
+  @Input() cheRunning: boolean;
```

Comment:
This appears to be unused

Reply:
used in template https://github.com/ashumilova/fabric8-ui/blob/4a59ebc3452e43902418fc118a462a4c98e39a4f/src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.html#L7

Reply:
Ah... I was expecting each component would use a similar Che input and the template would reference che.running.